src/index.html

```diff
@@ -11,7 +11,172 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>HTML Form</h1>
     <script type="text/javascript" src="./main.js"></script>
+    <form action="https://mate-academy-form-lesson.herokuapp.com/create-application" method="post">
+      <fieldset class="form-field">
+        <legend>Personal information</legend>
+        <label class="input">
+          Surname:
+          <input
+          type="text"
+          name="Surname"
+          required
+          autocomplete="off"
+          >
+        </label>
+
+        <label class="input">
+          Name:
+          <input
+          type="text"
+          name="Name"
```

Comment:
```suggestion
          name="name"
```
No need to capitalize the name value
![image](https://user-images.githubusercontent.com/96409563/224265148-e365cfc1-e6d6-4699-acea-7b6f79779637.png)